src/common/tests/conditions.js

```diff
@@ -4,7 +4,32 @@ export const elementValueIs = (element, text) =>
     new WebElementCondition('until element text is', () =>
         element.getAttribute('value').then(t => (t === text ? element : null)));
 
-export const elementIsClickable = element =>
-    new WebElementCondition('until element is clickable', () => (
-        element.isDisplayed() && element.isEnabled() ? element : null
-    ));
+export const inputElementIsFocusable = element =>
+    new WebElementCondition('until element is clicked', async () => {
+        const [isDisplayed, isEnabled] = await Promise.all([
+            element.isDisplayed(),
+            element.isEnabled(),
+        ]);
+
+        if (!isDisplayed || !isEnabled) return null;
+
+        return element
+                .sendKeys('')
+                .then(() => element)
+                .catch(() => null);
+    });
+
+export const elementIsClicked = element =>
+    new WebElementCondition('until element is clicked', async () => {
+        const [isDisplayed, isEnabled] = await Promise.all([
+            element.isDisplayed(),
+            element.isEnabled(),
+        ]);
+
+        if (!isDisplayed || !isEnabled) return null;
+
+        return element
+                .click()
+                .then(() => element)
+                .catch(() => null);
```

Comment:
Because selenium will throw a `WebDriverError` in all cases with only the message describing the exact prb. This message might change in future versions